BardMusicPlayer.Config/BmpConfig.cs

```diff
@@ -0,0 +1,49 @@
+﻿/*
+ * Copyright(c) 2021 MoogleTroupe, 2018-2020 parulina
+ * Licensed under the GPL v3 license. See https://github.com/BardMusicPlayer/BardMusicPlayer/blob/develop/LICENSE for full license information.
+ */
+
+using BardMusicPlayer.Common;
+using BardMusicPlayer.Config.JsonSettings.Autosave;
+
+namespace BardMusicPlayer.Config
+{
+    public class BmpConfig : JsonSettings.JsonSettings
+    {
+        private static BmpConfig _instance;
+        public BmpConfig() { }
+        public BmpConfig(string fileName) : base(fileName) { }
+
+        /// <summary>
+        /// This configuration's filename
+        /// </summary>
+        public override string FileName { get; set; }
+
+        /// <summary>
+        /// Initializes the config file
+        /// </summary>
+        /// <param name="filename">full path to the json config file</param>
+        public static void Initialize(string filename)
```

Comment:
moogle and i spoke about this here: https://discord.com/channels/476802720922206209/753359298842853446/839751802118340614